src/angular/processflow/step-header.scss

```diff
@@ -77,11 +67,9 @@
   }
 }
 
-.sbb-step-label-selected {
-  &::after {
-    left: 0;
-    width: 100%;
-  }
+.sbb-step-label-selected::after {
+  left: 0;
+  width: 100%;
 }
 
 .sbb-step-label-active.sbb-step-label-locked {
```

Comment:
.sbb-step-header:hover & {
    color: var(--sbb-color-text);
  }
it looks that in locked step the hover color has to be overwritten, therefore it should be color-text. or not?